src/results/outages.jl

```diff
@@ -12,6 +12,11 @@
 - `pv_to_storage_series_kw` Array of PV power sent to the battery in every outage modeled.
 - `pv_curtailed_series_kw` Array of PV curtailed in every outage modeled.
 - `pv_to_load_series_kw` Array of PV power used to meet load in every outage modeled.
+- `wind_microgrid_size_kw` Optimal microgrid PV capacity. Note that the name `PV` can change based on user provided `PV.name`.
```

Comment:
Update these descriptions to refer to Wind not PV

Reply:
Addressed in c6b049b515b7e07c4bd58fb4b9b31041ee31e1a7